src/plugins/nest.js

```diff
@@ -67,6 +68,34 @@ function createWrapCreateHandler (tracer, config) {
   }
 }
 
+function createWrapHandler (tracer, handler) {
+  let name = 'nestHandler'
+  if (handler.name) {
+    name = handler.name
+  }
+  const wrappedHandler = function () {
+    const scope = tracer.scope()
+    const childOf = scope.active()
+    const tags = { 'component': 'nest' }
+    if (name) {
+      tags['nest.callback'] = name
+    }
+    const span = tracer.startSpan(name, { childOf, tags })
+    try {
+      return handler.apply(this, arguments)
```

Comment:
Wouldn't it be desirable to activate the span for the handler?

Reply:
Yes, I missed that. Adding.